assets/src/js/search.js

```diff
@@ -1,137 +1,165 @@
 /* global localizations */
 
-// Search page.
-export const setupSearch = function($) {
-  const isSearch = !!$('body.search').length;
-  if (!isSearch) {
+const showHiddenRow = row => {
+  if (!row) {
     return;
   }
+  row.classList.remove('row-hidden');
+  row.style.display = 'block';
+};
 
-  const $search_form      = $( '#search_form' );
-  const $load_more_button = $( '.btn-load-more-click-scroll' );
-  let load_more_count   = 0;
-  let loaded_more       = false;
-
-  $( '#search-type button' ).click(function() {
```

Comment:
This seemed to be unused, I couldn't find any mention of this `search-type` id anywhere 🤷‍♀️